### PR TITLE
Fix windows plugins datetime

### PR DESCRIPTION
**Pull request by Zawadidone**
Fixes datetime fields of the usb and tasks plugin

### CODE REVIEW

File: dissect/target/plugins/os/windows/regf/usb.py [hunk 1]
```diff
@@ -19,10 +19,10 @@
         ("string", "product"),
         ("string", "version"),
         ("string", "friendlyname"),
-        ("string", "first_insert_date"),
-        ("string", "first_instal_date"),
-        ("string", "last_insert_date"),
-        ("string", "last_removal_date"),
+        ("datetime", "first_insert_date"),
+        ("datetime", "first_instal_date"),
```
Comment (Schamper) [inline dissect/target/plugins/os/windows/regf/usb.py:23]:
```suggestion
        ("datetime", "first_install_date"),
```

File: tests/test_plugins_os_windows_tasks.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
+from dissect.target.plugins.os.windows.tasks import TasksPlugin
+
+from ._utils import absolute_path
+
+
+def test_tasks(target_win, fs_win):
+    task_file = absolute_path("data/windows-task")
```
Comment (Schamper) [inline tests/test_plugins_os_windows_tasks.py:7]: Can you move the test data to `data/plugins/os/windows/tasks/` ?

**Review by Schamper — CHANGES_REQUESTED** [review]

File: dissect/target/plugins/os/windows/regf/usb.py [hunk 3]
```diff
@@ -19,10 +19,10 @@
         ("string", "product"),
         ("string", "version"),
         ("string", "friendlyname"),
-        ("string", "first_insert_date"),
-        ("string", "first_instal_date"),
-        ("string", "last_insert_date"),
-        ("string", "last_removal_date"),
+        ("datetime", "first_insert_date"),
```
[the remark this answers is not in the document]
Comment (Schamper) [reply dissect/target/plugins/os/windows/regf/usb.py:22]: Yeah let's do that. Can you also change it throughout the plugin?

[the remark this answers is not in the document]
Comment (Zawadidone) [reply dissect/target/plugins/os/windows/regf/usb.py]: Yes

**Review by Schamper — APPROVED** [review]

**Pull request merged**